<style>.resource{
        color:white;
    }</style> <md-content layout-padding class="main-container" style="overflow-x: hidden; height: 100%;
          background-color: #1f253d; color: #203e54"> <div class="profile-container"> <div class="main-container"> <profile-header></profile-header> </div> </div> <div class="md-title resource">Software for SDSU researchers</div> <p> “Bioinformatics Workbench”: Bioinformatics software at Math/Stat. Department Open to SDSU researchers The following bioinformatics software has been purchased through internal support: · Vector NTI Sequence Analysis v.11 (Invitrogen, Carlsbad, CA) for general sequence analysis; · Genomics Workbench (CLC bio LLC, Cambridge, MA) for analyzing and visualizing next-generation sequencing data; · Lasergene v.8 (DNASTAR, Madison, WI) for general sequence analyses and next-generation sequencing data analysis. Commercial statistics and data mining software: · JMP v.7 (SAS Institute, Cary, NC) for statistical modeling through a user-friendly graphical interface. · STATISTICA w/ Data Miner v.8 (StatSoft, Inc., Tulsa, OK) for statistical modeling and data mining · SAS Enterprise Miner v.5.3 (SAS Institute, Cary, NC) for data mining. Availability: Most of the software will be installed in the “Bioinformatics Workbench”, a high-end PC in Room 204, Harding Hall. All SDSU researchers/students are welcome to use it. Lasergene could be used from your own computer via network license. Please contact Dr Ge for a copy. </p> <div class="md-title resource">Tools and resources for analyzing biological data</div> <p> Useful websites for analysis of biological datasets NCBI and GenBank (Nucleotide, Protein, and Genome Database) GenBank is a protein and nucleic acid sequence database founded by the National Center for Biotechnology Information (NCBI) at the National Institute of Health. Containing over 100,000,000 sequences and continuing to grow this database is constructed from directly submitted sequences and is a key resource in homology and identification searches. Query sequences can be compared by using a BLAST search. NCBI BLAST: http://blast.ncbi.nlm.nih.gov/Blast.cgi BLAST Tutorial: http://www.ncbi.nlm.nih.gov/books/NBK1734/ GenBank can also be searched through a text-based inquiry such as a search for a gene, protein, or organism of interest. Using the Entrez search engine the query can generate such information as sequence data, taxonomy, publications on the subject, and three dimensional structures. Entrez Search: http://www.ncbi.nlm.nih.gov/sites/gquery You can also narrow your searches by choosing the database you wish to query from the Entrez search page. NCBI Nucleotide: http://www.ncbi.nlm.nih.gov/sites/entrez?db=nucleotide NCBI Protein: http://www.ncbi.nlm.nih.gov/protein NCBI Genome Search: http://www.ncbi.nlm.nih.gov/sites/entrez?db=genome NCBI has a very useful training and tutorial page that can be searched for help on any of the tools that the NCBI website contains. NCBI Training and Tutorials: http://www.ncbi.nlm.nih.gov/guide/training-tutorials/ Publications: NCBI BLAST: a better web interface: Nucl. Acids Res. (2008) 36(suppl 2): W5-W9 first published online April 24, 2008 Gene Expression Omnibus: NCBI gene expression and hybridization array data repository: Nucl. Acids Res. (2002) 30(1): 207-210 dbSNP: the NCBI database of genetic variation: Nucl. Acids Res. (2001) 29(1): 308-311 NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins: Nucl. Acids Res. (2007) 35(suppl 1): D61-D65 first published online November 27, 2006 Entrez Gene: gene-centered information at NCBI: Nucl. Acids Res. (2007) 35(suppl 1): D26-D31 first published online December 5, 2006 ---------------------------------------------------------- EMBL Nucleotide Sequence Database (Nucleotide, Protein, and Genome Database) http://www.ebi.ac.uk/embl/ EMBL is Europe’s nucleotide sequence database and works in collaboration with both GenBank and the DNA Database of Japan (DDBJ) to exchange sequence data. Text searches and sequence queries can be directed here: http://www.ebi.ac.uk/ena/ For more complex searches the use of the Sequence Retrieval System server is recommended and serves a similar function as NCBI’s Entrez search by allowing inquiries across several different databases. Sequence Retrieval System server: http://srs.ebi.ac.uk/ Publications: A new bioinformatics analysis tools framework at EMBL–EBI: Nucl. Acids Res. (2010) 38(suppl 2): W695-W699 first published online May 3, 2010 The European Nucleotide Archive: Nucl. Acids Res. (2011) 39(suppl 1): D28-D31 first published online October 23, 2010 Priorities for nucleotide trace, sequence and annotation data capture at the Ensembl Trace Archive and the EMBL Nucleotide Sequence Database: Nucl. Acids Res. (2008) 36(suppl 1): D5-D12 first published online November 26, 2007 EMBL Nucleotide Sequence Database in 2006: Nucl. Acids Res. (2007) 35(suppl 1): D16-D20 first published online December 5, 2006 Cochrane G.R. and Galperin M.Y. (2010)The 2010 Nucleic Acids Research Database Issue and online Database Collection: a community of data resources. Nucleic Acids Research 38: D1-D4. Leinonen R., Akhtar R., Birney E, Bonfield J., Bower L., Corbett M., Cheng Y., Demiralp F., Faruque N., Goodgame N., Gibson R, Hoad G., Hunter C., Jang M., Leonard S., Lin Q., Lopez R., Maguire M., McWilliam H., Plaister S, Radhakrishnan R, Sobhany S, Slater G., Ten Hoopen P., Valentin F., Vaughan R., Zalunin V., Zerbino D., Cochrane G. (2010) Improvements to services at the European Nucleotide Archive. Nucleic Acids Research 38: D39-D45. Shumway M., Cochrane G., Sugawara H. (2010)Archiving next generation sequencing data. Nucleic Acids Research 38: D870-D871. Cochrane G., Akhtar R., Bonfield J., Bower L., Demiralp F., Faruque N., Gibson R., Hoad G., Hubbard T., Hunter C., Jang M., Juhos S., Leinonen R., Leonard S., Lin Q., Lopez R., Lorenc D., McWilliam H., Mukherjee G., Plaister S., Radhakrishnan R., Robinson S., Sobhany S., Hoopen P.T., Vaughan R., Zalunin V., Birney E. (2009)Petabyte-scale innovations at the European Nucleotide Archive. Nucleic Acids Research 37: D19-D25. Galperin M.Y., Cochrane G.R. (2009)Nucleic Acids Research annual Database Issue and the NAR online Molecular Biology Database Collection in 2009. Nucleic Acids Research 37: D1-D4. Angiuoli S.V., Gussman A., Klimke W., Cochrane G., Field D., Garrity G., Kodira C.D., Kyrpides N., Madupu R., Markowitz V., Tatusova T., Thomson N., White O. (2008). Toward an Online Repository of Standard Operating Procedures (SOPs) for (Meta)genomic Annotation. OMICS 12(2): 137-141. Cochrane G., Akhtar R., Aldebert P., Althorpe N., Baldwin A., Bates K., Bhattacharyya S., Bonfield J., Bower L., Browne P., Castro M., Cox A., Demiralp F., Eberhardt R., Faruque N., Hoad G., Jang M., Kulikova T., Labarga A., Leinonen R., Leonard S., Lin Q., Lopez R., Lorenc D., McWilliam H., Mukherjee G., Nardone F., Plaister S., Robinson S., Sobhany S., Vaughan R., Wu D., Zhu W., Apweiler R., Hubbard T., Birney E. (2008) Priorities for nucleotide trace, sequence and annotation data capture at the Ensembl Trace Archive and the EMBL Nucleotide Sequence Database. Nucleic Acids Research 36: D5-D12. ------------------------------------------------------------ DNA Data Bank of Japan (Nucleotide, Protein, and Genome Database) http://www.ddbj.nig.ac.jp/ Japanese database for sequence data that works in much the same way as NCBI’s GenBank and EMBL. Publications: Biological Databases at DNA Data Bank of Japan in the Era of Next-Generation Sequencing Technologies: ADVANCES IN COMPUTATIONAL BIOLOGY. Advances in Experimental Medicine and Biology, 2011, Volume 680, Part 1, 125-135 DDBJ dealing with mass data produced by the second generation sequencer: Nucl. Acids Res. (2009) 37(suppl 1): D16-D18 first published online October 16, 2008 DNA Data Bank of Japan (DDBJ) in XML: Nucl. Acids Res. (2003) 31(1): 13-16 ---------------------------------------------------------------- GeneCards: The Human Gene Compendium (Human Gene Search) http://www.genecards.org/ A searchable database for information on human genes. The search mines for data from a variety of resources to provide comprehensive information on the gene in an easy to understand and navigate format. Publications: Harel A, Dalah I, Pietrokovski M, Safran M and Lancet D. Omics Data Management and Annotation, Chapter 3 Bioinformatics for Omics Data: Methods and ProtocolsSeries: Methods in Molecular Biology, Vol. 719, ISBN: 978-1-61779-026-3, Mayer, Bernd (Ed.), Humana Press, Due: March 29, 2011 Safran M, Dalah I, Alexander J, Rosen N, Iny Stein T, Shmoish M, Nativ N, Bahir I, Doniger T, Krug H, Sirota-Madi A, Olender T, Golan Y, Stelzer G, Harel A and Lancet D. GeneCards Version 3: the human gene integrator Database 2010; doi: 10.1093/database/baq020 Stelzer G, Inger A, Olender T, Iny Stein T, Dalah I, Harel A, Safran M and Lancet D. GeneDecks: paralog hunting and gene-set distillation with GeneCards annotation OMICS, 2009, 13(6) Harel A, Inger A, Stelzer G, Strichman-Almashanu L, Dalah I, Safran M and Lancet D. GIFtS: annotation landscape analysis with GeneCards BMC Bioinformatics 2009, 10:348 Stelzer G, Harel A, Inger A, Olender T, Dalah I, Shmoish M, Ron S, Safran M and Lancet D. GeneDecks: GeneCards-based paralog hunting and gene set distillation.CSB2008 Aug, 2008 Stelzer G, Harel A, Dalah A, Rosen N, Shmoish M, Iny Stein T, Sirota A, Madi A, Safran M and Lancet D. GeneCards: One stop site for human gene research FISEB (ILANIT), Jan, 2008 --------------------------------------------------------------------- Mobyle at the Pasteur Institute (Bioinformatics Tools and Software) http://mobyle.pasteur.fr/cgi-bin/portal.py?#welcome Web interface software that allows access to over 200 bioinformatics programs without installation on a local computer. Contains programs targeting: Alignment Assembly Database Display Genetics HMM (Hidden Markov Model) Nucleic Phylogeny Protein Sequence Structure Publication: Bertrand Néron, Hervé Ménager, Corinne Maufrais, Nicolas Joly, Julien Maupetit, Sébastien Letort, Sébastien Carrere, Pierre Tuffery, and Catherine Letondal. Mobyle: a new full web bioinformatics framework. Bioinformatics (2009) 25(22): 3005-3011 first published online August 17, 2009 ------------------------------------------------------------------ Softberry (Bioinformatics Tools and Software) http://linux1.softberry.com/berry.pthml Has downloadable bioinformatics software for Windows, Linux, and Mac systems. Available software in the following analysis categories: Automatic genome annotation Analysis of gene regulation Analysis of expression data Sequence comparison Protein 3D-structure analysis and modeling Analysis of protein functions --------------------------------------------------------------- ExPASy Proteomics Tools (Protein analysis tools and software) http://www.expasy.ch/tools/ Proteomics data analysis tools. Some are available for use on a web server while others need to be downloaded. Tool categories include: Protein identification and characterization DNA -> Protein translation Similarity searches Pattern and profile searches Post-translational modification prediction Topology prediction Primary structure analysis Secondary structure prediction Tertiary structure tools Quaternary structure tools Molecular modeling and visualization tools Prediction of disordered regions Sequence alignment Phylogenetic analysis Biological text analysis Statistical tools ------------------------------------------------- The Sequence Manipulation Suite (Nucleic acid and protein sequence tools) http://www.bioinformatics.org/SMS/index.html Web based programs that can be used for analysis and formatting DNA and protein sequences. ---------------------------------------------------- GOToolBox (Bioinformatics for gene classification) http://genome.crg.es/GOToolBox/ A web server that contains a series of programs that allow a functional analysis of groups of genes based on the Gene Ontology Project (http://www.geneontology.org/). Publication: Martin D, Brun C, Remy E, Mouren P, Thieffry D, Jacq B. GOToolBox : functional investigation of gene datasets based on Gene Ontology. Genome Biology 2004;5(12):R101. Epub 2004 Nov 26. --------------------------------------------------- Online Analysis Tools for Bioinformatics (Bioinformatics Tools) http://molbiol-tools.ca Excellent resource for software and online programs targeting biological data analysis. Includes software on: Carbohydrates DNA Sequence Analysis Genomics Protein Sequence Analysis RNA Sequence Analysis Transcription Analysis -------------------------------------------------- Mol-Bio Analysis Software (Bioinformatics Tools) http://bomi.ou.edu/russell/mol-bio.html Another great resource for molecular biology analysis software and webtools. Contains tools for: Multipurpose software packages Sequence search Sequence alignment Phylogeny Transcription elements Gene prediction Gene expression PCR related Oligonucleotide structure Oligonucleotide repeats Misc. Oligonucleotide tools Translation Protein Secondary Structure Protein Tertiary Structure Protein Domain/Motif Protein Modification Misc. Protein Tools --------------------------------------------------------- Genomics and Bioinformatics Group (Cancer research bioinformatics tools) http://discover.nci.nih.gov Bioinformatics software packages and data analysis tools. These tools are targeting cancer research. -------------------------------------------------------- KEGG (Kyoto Encyclopedia of Genes and Genomes) (Tools for gene and biochemical pathway characterization) http://www.kegg.com KEGG is a widely used reference database for biological interpretation of large datasets generated by high-throughput technologies such as pyrosequencing, metagenomics, and microarray data. It can be used to search genomes, characterize biochemical pathways, and map microarray data. It’s three main categories of databases include systems information, genomic information, and chemical information. Publications: KEGG for linking genomes to life and the environment. Nucl. Acids Res. (2008) 36(suppl 1): D480-D484 first published online December 12, 2007 KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucl. Acids Res. (2000) 28(1): 27-30 From genomics to chemical genomics: new developments in KEGG. Nucl. Acids Res. 34(suppl 1): D354-D357 The KEGG resource for deciphering the genome. Nucl. Acids Res. (2004) 32(suppl 1): D277-D280 ---------------------------------------------------------------- DAVID (Database for Annotation, Visualization, and Integrated Discovery) (Microarray and gene characterization tools) http://david.abcc.ncifcrf.gov/ Provides a useful and comprehensive set of functional tools to understand the biological meanings behind large gene lists. Very useful for analysis of microarray data. It’s main tools of interest are the functional annotation tool which retrieves information about the queried genes from a number of different databases and the gene functional classification tool which reduces lists of genes into functionally related groups to try and reveal the biological meaning in the data. Publications: Systematic and integrative analysis of large gene lists using DAVID Bioinformatics Resources. Nature Protoc. 2009;4(1):44-57. Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. Nucleic Acids Res. 2009 Jan;37(1):1-13. DAVID gene ID conversion tool. Bioinformation. 2008. July; 2(10): 428-430 Stephens R, Baseler MW, Lane HC, Lempicki RA. DAVID Bioinformatics Resources: Expanded annotation database and novel algorithms to better extract biology from large gene lists. Nucleic Acids Res. 2007 Jul 1;35. ------------------------------------------------------------ Reactome (Biological Pathways database) http://www.genomeknowledge.org An online resource that contains a curated biological pathway database for human and a small number of other well characterized organisms. Publications: Reactome knowledgebase of human biological pathways and processes. Nucl. Acids Res. (2009) 37(suppl 1): D619-D622 first published online November 3, 2008 Reactome: a knowledgebase of biological pathways. Nucl. Acids Res. (2005) 33(suppl 1): D428-D432 Reactome: a database of reactions, pathways and biological processes. Nucleic Acids Res. 2011 Jan;39(Database issue):D691-7. ------------------------------------------------------------ UCSC Genome Bioinformatics (Sequenced genomes for Eukaryotes, Archaea, and Bacteria) http://genome.ucsc.edu This university page and server contains a large number of sequenced genomes including the human genome, other eukaryotic organisms, as well as a large number of prokaryotic microbial genomes and archaeal genomes. Publications: The UCSC Genome Browser database: update 2010. Nucl. Acids Res. (2010) 38(suppl 1): D613-D619 first published online November 11, 2009 The Human Genome Browser at UCSC. Genome Res. June 1, 2002 12: 996-1006; Published in Advance May 16, 2002 The UCSC Genome Browser Database. Nucl. Acids Res. (2003) 31(1): 51-54 --------------------------------------------------------------------- Galaxy (Bioinformatics tools, Metagenomic and Genomic Analysis) http://main.g2.bx.psu.edu An online multipurpose software package for biological analysis tools and data integration. It can analyze multiple alignments, compare genomic annotations, profile metagenomic samples, and has a wide variety of other functions. There is a wiki site available to tutorials and troubleshooting. https://bitbucket.org/galaxy/galaxy-central/wiki/Home Publications: Goecks, J, Nekrutenko, A, Taylor, J and The Galaxy Team. Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences. Genome Biol. 2010 Aug 25;11(8):R86. Blankenberg D, Von Kuster G, Coraor N, Ananda G, Lazarus R, Mangan M, Nekrutenko A, Taylor J. "Galaxy: a web-based genome analysis tool for experimentalists". Current Protocols in Molecular Biology. 2010 Jan; Chapter 19:Unit 19.10.1-21. ---------------------------------------------------------------------- The Gene Ontology (Characterization of gene and groups of genes) http://www.geneontology.org/ The main site is the original source for standardization of gene product characteristics and gene product annotation data that is used across a wide variety of databases to classify genes into functional groups and try to fit biological processes together. The site also offers links to a variety of bioinformatics tools that integrate gene ontology tools into their software functions. Microarray Data http://www.geneontology.org/GO.tools.microarray.shtml Search Engines http://www.geneontology.org/GO.tools_by_type.search.shtml Protein Interaction http://www.geneontology.org/GO.tools_by_type.protein_interaction.shtml Publication: The Gene Ontology Consortium. Gene ontology: tool for the unification of biology. Nat. Genet.. May 2000;25(1):25-9. -------------------------------------------------------------------- Ribosomal Database Project (16S rRNA Database) http://rdp.cme.msu.edu/ The Ribosomal Database Project is a database for ribosomal sequences and provides services such as online data analysis and aligned 16S rRNA bacterial and archaeal sequences. RDP is an excellent database for high quality bacterial and archaeal sequence data. Individual sequences or entire sequence libraries can be classified. Sequence libraries can also be compared to each other and phylogenetic trees built from database and personal sequences. Has a very useful pyrosequencing pipeline for high-througput data processing and analysis. Publications: Cole, J. R., Q. Wang, E. Cardenas, J. Fish, B. Chai, R. J. Farris, A. S. Kulam-Syed-Mohideen, D. M. McGarrell, T. Marsh, G. M. Garrity, and J. M. Tiedje. 2009. The Ribosomal Database Project: improved alignments and new tools for rRNA analysis. Nucleic Acids Res. 37 (Database issue): D141-D145 Cole, J. R., B. Chai, R. J. Farris, Q. Wang, A. S. Kulam-Syed-Mohideen, D. M. McGarrell, A. M. Bandela, E. Cardenas, G. M. Garrity, and J. M. Tiedje. 2007. The ribosomal database project (RDP-II): introducing myRDP space and quality controlled public data. Nucleic Acids Res. 35 (Database issue): D169-D172 If you use RDP Classifier, please cite: Wang, Q, G. M. Garrity, J. M. Tiedje, and J. R. Cole. 2007. Naïve Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy. Appl Environ Microbiol. 73(16):5261-7. -------------------------------------------------------------------- UniProt (Protein Database) http://www.uniprot.org/ UniProt is to provides comprehensive, high-quality and freely accessible protein sequence and functional information. UniProt can be searched by use of text or BLAST searches for identification of proteins. Publications: The UniProt Consortium. Ongoing and future developments at the Universal Protein Resource. Nucleic Acids Res. 39: D214-D219 (2011). Jain E., Bairoch A., Duvaud S., Phan I., Redaschi N., Suzek B.E., Martin M.J., McGarvey P., Gasteiger E. Infrastructure for the life sciences: design and implementation of the UniProt website. BMC Bioinformatics, 10:136 (2009). ---------------------------------------------------------------------------------- Protein Information Resource (Protein Database) http://pir.georgetown.edu/pirwww/index.shtml The Universal Protein Resource (UniProt) provides a single, centralized resource for protein sequences and functional information. A member of the UniProt consortium, this site allows a number of different types of protein searches including BLAST/FASTA, related sequences, peptide match, and pattern searches. Search Tools: http://pir.georgetown.edu/pirwww/search/searchtools.shtml Publications: Protein-centric data integration for functional analysis of comparative proteomics data. McGarvey PB, Zhang J, Natale DA, Wu CH, Huang H. Methods Mol Biol. 694:323-39. 2011. Protein bioinformatics databases and resources. Chen C, Huang H, Wu CH. Methods Mol Biol. 694:3-24. 2011. Framework for a protein ontology. Natale DA, Arighi CN, Barker WC, Blake J, Chang TC, Hu Z, Liu H, Smith B, Wu CH. BMC Bioinformatics. 8 Suppl 9:S1. 2007. ------------------------------------------------------------------------------------------- SWISS-PROT (Protein Database) http://us.expasy.org/sprot/ This database is composed of protein sequences and provides a high level of annotation, quality, and a reduction of redundancy. Publications: The Gene Ontology Annotation (GOA) Project: Implementation of GO in SWISS-PROT, TrEMBL, and InterPro. Genome Res. April 1, 2003 13: 662-672; Published in Advance March 12, 2003 The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003. Nucl. Acids Res. (2003) 31(1): 365-370 ---------------------------------------------------------------------------------------------- Worldwide Protein Data Bank (Protein Database) http://www.wwpdb.org/ Functions as a deposition, data processing, and distributor of protein databank information. Protein data can be downloaded from ftp servers, protein data can be deposited, and structural information on proteins can be searched for. --------------------------------------------------------- Greengenes (16S rRNA Database) http://greengenes.lbl.gov/cgi-bin/nph-index.cgi Provides access to current and comprehensive 16S rRNA sequence alignment for browsing, blasting, probing, and downloading. These tools are useful for design of probes, microarray result analysis, and alignment/annotation of new sequences. The database information is quality controlled and provides useful tools for analysis of prokaryotic sequences from environmental or clinical samples. Publications: General Greengenes Database and Tools Citation: DeSantis, T. Z., P. Hugenholtz, N. Larsen, M. Rojas, E. L. Brodie, K. Keller, T. Huber, D. Dalevi, P. Hu, and G. L. Andersen. 2006. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB. Appl Environ Microbiol 72:5069-72. NAST Alignment Tool Citation: DeSantis, T. Z., P. Hugenholtz, K. Keller, E. L. Brodie, N. Larsen, Y. M. Piceno, R. Phan, and G. L. Andersen. 2006. NAST: a multiple sequence alignment server for comparative analysis of 16S rRNA genes. Nucleic Acids Res 34:W394-9. Tree Management Citation: Dalevi D, T. Z. DeSantis, J. Fredslund, G. L. Andersen, V. M. Markowitz, P. Hugenholtz P. 2007. Automated group assignment in large phylogenetic trees using GRUNT: GRouping, Ungrouping, Naming Tool. BMC Bioinformatics. Oct 18;8(1):402 Original Publication DeSantis, T. Z., I. Dubosarskiy, S. R. Murray, and G. L. Andersen. 2003. Comprehensive aligned sequence construction for automated design of effective probes (CASCADE-P) using 16S rDNA. Bioinformatics 19:1461-8. ------------------------------------------------------------------ SILVA rRNA Database (RNA sequence database) http://www.arb-silva.de/ This database provides quality checked and regularly updated datasets of small (16S/18S, SSU) and large (23S/28S, LSU) subunit ribosomal RNA sequences for all three domains of life (Bacteria, Archaea, and Eurkarya). Publications: Pruesse, E., C. Quast, K. Knittel, B. Fuchs, W. Ludwig, J. Peplies, and F. O. Glöckner. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. Nuc. Acids Res. 2007; Vol. 35, No. 21, p. 7188-7196 ------------------------------------------------------------------- Nucleic Acid Database (3D structure of Nucleic Acids) http://ndbserver.rutgers.edu/ This database is a repository of 3D structural information about nucleic acids. The NDB atlas gives summary information and images of nucleic acids characterized by both x-ray crystallography and NMR. The search engine can be used to find molecules of interest and the deposited data can be downloaded for personal use. Publications: H. M. Berman, W. K. Olson, D. L. Beveridge, J. Westbrook, A. Gelbin, T. Demeny, S.-H. Hsieh, A. R. Srinivasan, and B. Schneider. (1992) The Nucleic Acid Database: A Comprehensive Relational Database of Three-Dimensional Structures of Nucleic Acids. Biophys. J., 63, 751-759. ---------------------------------------------------------------------- Scientific Article and Journal Search Engines PubMed: http://www.ncbi.nlm.nih.gov/pubmed Web of Science: http://apps.isiknowledge.com GoogleScholar: http://scholar.google.com BioText Search Engine: http://biosearch.berkeley.edu Scopus: http://www.scopus.com/home.url </p> <div class="md-title resource">Tutorial & help websites for bioinformatics software and analysis</div> <div class="md-title resource">Bioinformatics Cheat Sheet</div> <p> Bioinformatics cheat sheet 1. Yes you can write a small Perl or R script to do data analyses. There are thousands of nerds contributing free program packages: Bioconductor for use with R, and BioPerl with Perl. Perl Express is a IDE. 2. Excel is powerful. Excel can read plain text files. Select delimiters while opening the file or use the powerful Text to Column function under Data. Hold the control key, your cursor goes a long way; F4 key fixes your cell in formula. You can sort the columns too. Filters can be used on both numerical and text columns. Combine multiple cells “=A1&B1”. Functions: sum, average, stdev, min, max, ttest, rand (random number 0 -1), sumif and countif (conditional sum and count); vlookup, left (return the left n character of a string), right, len (length) ; Full list of functions. RExcel enables using R from Excel. Array formulas, pivot table. Databases 1. FASTA is a simple format for storing DNA and protein sequences. GenBank is another format with annotation. Conversion between formats is easy. For example this web site: http://www-bimas.cit.nih.gov/molbio/readseq/ 2. GenBank accession number is a stable ID for a sequence record. Some GenBank entries have error. 3. A gene can be identified by many IDs, including official gene symbol (TP53), RefSeq (NM_000546), Entrez gene ID (7157), ENSEMBL ID (ENSG00000141510), UniGene Cluster ID (Hs.654481). For some poorly annotated genes, the only ID available might be a GenBank accession number of the associated mRNA (AF052180) or Expressed sequence tag (EST, i.e. BX111932). More IDs. Some tools can help convert between IDs, i.e., Source at Stanford or DAVID at NIH. 4. NCBI website: data archive, expert annotation/knowledge, software tools. World’s most comprehensive genomics databases. Nucleotide, Protein, PubMed, Gene, gene expression, genetic variation. Ensembl is a European version of comprehensive database. 5. Gene-centric database: other than NCBI Gene, GeneCards is a very useful resource. 6. For plants, web sites like www.plantGDB.org, www.ARABIDOPSIS.org, www.GRAMENE.org/ host more information. 7. ExPASy holds protein information resource managed by Swiss Institute of Bioinformatics. 8. Genome Browsers enable access to whole genome sequences, integrated with gene annotations, mRNA sequences, genetic variations, etc. UCSC genome browser is one of the best for mammalian. Others include Ensembl genome browser, NCBI map viewer, plantGDB, microbe genome browser, flybase, WormBase etc.. Sequence Analyses 1. There are three major strategies for comparing nucleic sequences: Dot matrix, dynamic programming and word-based methods. 2. Dot matrix method enable visualization of similarity. It can detect insertion, deletion, inversion, repeats in a pair of sequences or between a sequence and itself. DotMatcher is one of the programs. 3. Global alignment attempts to compare two sequences for the whole length, while local alignment focuses on identifying small blocks of shared sequences. Increase gap penalty will favor local alignment. 4. Dynamic programming can provide optimal alignment between two sequences. It can produce both local and global alignments. Needleman-Wunsch algorithm for global; Smith-Waterman algorithm for local. LAIGN. 5. Percent Identity: # identical amino acid pairs divided by length of alignment; Percent Similarity: # of identical or similar amino acid pairs divided by length of alignment. 6. PAM matrix assumes Markov model of evolution and is more suited for study evolution. BLOSUM matrix is suited by discovering conserved domains, but is in general more widely used. BLOSUM62 is the default of BLAST. 7. Weird things do happen just by chance, especially when something is tried over and over again. Local alignment scores of random sequences follow an extreme value distribution. This forms the basis for theoretical evaluation of P values of an alignment. Other strategies include shuffling one sequences many times and observe score distribution. 8. CLUSTALW is not the only program that does multiple sequence alignment. It is a progressive method offers faster speed. Other programs use iterative methods that might find subtle similarities DIALIGN, POA. 9. Phylogenetic tree displays the relationship between the similarities of set of sequences. Neighbor-joining is one of the methods for producing phylogenetic tree. PHYLIP package. Bootstrapping on a subset of nucleic positions can be used to assess reliability of the tree. 10. MEME can detect shared short sequence motifs in a set of sequences. Other methods include Gibbs Sampler and Hidden Markov Models. 11. BioEdit is one of many free sequence editors and viewers. Commercial versions such as Vector NTI, LaserGene, CLC Bio Sequence Viewer are available. 12. Comprehensive suite of software for sequence analysis exits. Biology Workbench includes can carry out most sequence analyses on the web. EMBOSS can be downloaded as a Java program. 13. Scan your protein sequence for known domains can be done with SMART or PFAM database, among others. These databases contain lists of known domains saved as Hidden Markov Models. 14. BLAST uses word-based strategy to quick search for similarity. a. Select the appropriate database to compare: Genome, well annotated sequences (RefSeq), mRNAs, ESTs (spliced & unspliced), etc. b. BLAST is more than one program. BLASTn, BLASTp, BLASTx. c. Protein searches are more specific than DNA searches. Protein-protein, DNA-protein searches. d. E value of 1 assigned to a hit can be interpreted as meaning that in a database of the current size one might expect to see 1 match with a similar score simply by chance. Short queries have relatively high E values (even bigger than 1) that can be significant. Longer queries (>100bp) E values should be smaller than 10-20. You can change the Expect value threshold on most BLAST search pages. When the Expect value is increased from the default value of 10, a larger list with more low-scoring hits can be reported. e. Low complexity sequences like “AAATAAAAAAAATAAAAAAT” should be filtered out to avoid artificially significant E values. Repeated sequences can also be filtered out using a database of repeated sequences. f. BLAST program is easy to download and install on your own computer for local queries. g. Searching with short sequences needs different set of parameters. h. In general use default settings and parameters, unless you have a reason to change. i. There are BioPerl and BioPython programs designed to parse and process large output files of BLAST. 15. BLAST is not the only program for database searching. It is fast; but it might not be as sensitive as FASTA program or dynamic programming based methods like SSEARCH or GGSEARCH. 16. If you want to map a sequence against genome sequence, BLAT is the program of choice. BLAT has is not a brother of BLAST. NCBI BLAST can also do genome mapping. DNA microarray 1. DNA microarray enable detection of how many copies a gene is transcribed into RNA. High throughput (thousands of genes) and low cost. 2. The two fundamental principles underlying the technology is (a) DNA molecules are more stable as complementary double strands so that we can use one strand as a sensor, and (b) short piece of DNA can identify a gene. 3. DNA microarrays rely on CCD devices to capture images; hence comes its inherent noise at the lower end of its dynamic range. 4. While cDNA microarrays use two biological samples differentially dyed to obtain ratios, oligonucleotide microarrays only analyze one biological sample (hence one color) per experiment. The latter has been more widely used. Affymetrix microarray is the dominant platform. 5. Typical steps in analyzing DNA microarray data: Pre-processing, array image quality control, normalization, annotation of probes, variation filtering, hierarchical clustering, selection of differentially expressed genes, and pathway analyses/Gene Ontology. 6. Hierarchical clustering is perhaps the most widely used method to analyze and visually present DNA microarray data. It represents a data point in a matrix by color and then shuffles the rows and columns so that similar genes/samples appear together. Hierarchical clustering is a powerful tool for visualizing any matrix of data. 7. Cluster and Treeview are a pair of programs that can be used to do hierarchical clustering. Later versions of Cluster 3.0 and Java Treeview are more stable. a. Input file to Cluster need to be in tab-delimited text file; the second column can contain gene names, but the header must be named as “NAME”. b. After loading input files, double check the numbers of rows and columns. c. If file has been correctly loaded, we normally will go to “Adjust Data” which enable us to do Log Transformation if the data is not transformed, and we will often need to Median Center: Genes. d. We typically go to “Hierarchical Clustering” and select Cluster under both Genes and Arrays and use “Average Linkage Clustering” since it is more robust. 8. For most microarray data, each data point (gene X in sample #3) has a reliability indicator (Detection P value ) which indicates how reliable the expression score is. Since we do not expect all genes in the genome to be expressed in any specific cell, it is no surprise that 60% of the genes does not produce signal that are significantly higher than random background. 9. It is OK to remove these non-expressed genes from your dataset before running statistical tests. 10. Student’s T test is not appropriate for analyzing DNA microarrays where only a few (3 vs. 3) replicates are available. The variance cannot be reliably detected due to small sample size. If N is bigger (>30) , then T test is OK. 11. Regularized T test like Cyber-T or Significant Analyses of Microarray (SAM) can be used to obtain more reliable tests. 12. Multiple testing is a serious problem in analyzing DNA microarray data. False Discovery Rate (FDR) can be used. qvalue package in R. 13. Permutation test can be carried out to test if there is any significant difference in the two biological samples in terms of gene expression. GenePattern program. 14. Limma package in Bioconductor enables analyses of DNA microarray data for more complicated factor designs. 15. Many methods exist to identify over-represented gene categories in a set of selected genes. Pathway analysis. NIH DAVID. Onto-express. </p> <div class="md-title resource">Presenting</div> <div class="md-title resource">Presenting</div> <div class="md-title resource">Presenting</div> <div class="md-title resource">Presenting</div> <div class="md-title resource">Presenting</div> </md-content>